tests/Integration/OptionsTest.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * Integration Tests: Plugin options
```

Comment:
I think this is not an integration test.

Reply:
How so? We're using the `register_post_type` and `unregister_post_type` WordPress functions in one of the tests. I don't feel that there's a need to break this file into unit and integration tests, but let me know if you do.